clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -333,6 +379,17 @@ private EhcacheEntityResponse validate(ValidateCacheManager message) {
    * @return an {@code EhcacheEntityResponse} indicating the success or failure of the store creation operation
    */
   private EhcacheEntityResponse createServerStore(ClientDescriptor clientDescriptor, CreateServerStore createServerStore) {
+    ClientState clientState = this.clientStateMap.get(clientDescriptor);
+    if (clientState == null) {
+      return failure(new IllegalStateException("Client " + clientDescriptor + " is not connected to the Clustered Store Manager"));
+    }
+    if (!clientState.isAttached()) {
+      return failure(new IllegalStateException("Client " + clientDescriptor + " is not attached to the Clustered Store Manager"));
+    }
+    if (!isConfigured()) {
+      return failure(new IllegalStateException("Clustered Store Manager is not configured"));
```

Comment:
This exception is actually transmitted back to the requesting client making the client ID somewhat redundant.  (It really didn't need to be provided in the exception messages above it.)  We should probably log these in the server before returning them and the client ID _needs_ to be in the log messages.

In general, we need  to "normalize" server- and client-side logging and exception messages to include information needed to track fault contributions back to clients (in particular).

I'll adjust this one (and the others like it) as an interim move until we take care of the more general logging/exception issue.